templates/zerver/api/changelog.md

```diff
@@ -0,0 +1,78 @@
+# Changelog
+
+This page documents changes to the Zulip Server API over time.
+
+The recommended way for a client like the Zulip mobile or desktop apps
+that needs to support interaction with a wide range of different Zulip
+server versions is to check the `zulip_feature_level` parameter in the
+`/register` and `/server_settings` responses to determine which of the
+below features are supported.
+
+## Changes in Zulip 2.2
+
+**Not yet stabilized**:
+
+* [`POST /register`](/api/register-queue): Added `slim_presence`
+  parameter.  Still under development.
+
+**Feature level 2**:
+
+* [`POST /messages/{message_id}/reactions`](/api/add-emoji-reaction):
+  The `emoji_type` parameter is optional; the server will guess the
+  emoji_type if it is not specified (checking custom emoji, then
+  unicode emoji for any with the provided name).
```

Comment:
This change isn't documented on the endpoint's page.

Reply:
Yeah, adding that in a next commit.